simpledisplay.d

```diff
@@ -1536,7 +1536,9 @@ void setOpenGLContextVersion() (ubyte hi, ubyte lo) { sdpyOpenGLContextVersion =
 		Added April 24, 2023  (dub v11.0)
 +/
 auto openGLCurrentContext() {
-	version(Windows)
+	version(without_opengl)
+		return null;
```

Comment:
is returning `null` a good idea? or should we version the whole function?

Reply:
ummm probably the whole function, you have no reason to ever call the function at all if you aren't wanting any opengl at all